### PR TITLE
docs: Home page

### PR DESCRIPTION
**Pull request by turdiyev**

### CODE REVIEW

**Comment by avocadowastaken** [comment]
Next time assign me as a reviewer :) i got lost in tons of emails from github.

File: TRANSLATION.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# reactjs.org saytining O'zbek tilidagi tarjimasi
```
Comment (avocadowastaken) [inline TRANSLATION.md]: How this file end up here again? Try to sync your fork https://help.github.com/articles/syncing-a-fork/

**Review by avocadowastaken — CHANGES_REQUESTED** [review]
Rebase you fork pls

**Comment by avocadowastaken** [comment]
@Shoyadbek please review this

File: TRANSLATION.md (hunk 1 above)
Comment (turdiyev) [reply TRANSLATION.md]: hm, i synced master only

**Comment by turdiyev** [comment]
> Rebase you fork pls

Ok, I merged

**Review by avocadowastaken — APPROVED** [review]

**Pull request merged**